cpu/amd64/asm.go

```diff
@@ -28,6 +28,26 @@ type (
 func (op aad) AppendAMD64(b []byte) []byte { return append(b, 0xD5, byte(op)) }
 func (op aam) AppendAMD64(b []byte) []byte { return append(b, 0xD4, byte(op)) }
 
+//asm:AAA
+func Aaa() literal { return aaa }
```

Comment:
The exported functions are in cpu/amd64/api.go not here, this file shouldn't export anything, only implement the amd64.Appender